docs/template/public/less/_side-nav.less

```diff
@@ -42,10 +42,11 @@
        
         &:hover {
             color: @blue_gd;
+            background-color: transparent;
         }
     }
 
-    .kss-active a {
```

Comment:
@HybrisRawls the class names was changed because that's the class bootstrap's scroll spy component adds and removes to keep tracking of the active node when scrolling.